invenio_app_ils/records/api.py

```diff
@@ -71,23 +71,21 @@ def get_record_by_pid(cls, pid, with_deleted=False, pid_type=None):
     def create(cls, data, id_=None, **kwargs):
         """Create IlsRecord record."""
         data["$schema"] = current_jsonschemas.path_to_url(cls._schema)
-        getattr(cls, "validate_create", lambda x: x)(data)
         return super(IlsRecord, cls).create(data, id_=id_, **kwargs)
 
     def clear(self):
         """Clear IlsRecord data."""
         super(IlsRecord, self).clear()
         self["$schema"] = current_jsonschemas.path_to_url(self._schema)
 
-    def patch(self, patch):
-        """Validate patch metadata."""
-        getattr(self, "validate_patch", lambda x: x)(patch)
-        return super(IlsRecord, self).patch(patch)
+    def validate(self, **kwargs):
```

Comment:
Nice job and catch! @ntarocco @topless @kprzerwa do you think that we will need validation per method or one common would be fine?